vignettes/wrapping-apis.Rmd

```diff
@@ -213,92 +224,130 @@ faker_person <- function(gender = NULL, birthday_start = NULL, birthday_end = NU
 faker_person("male", quantity = 5)
 ```
 
-The next steps would be to export and document this function; but I'll leave that up to you.
+The next steps would be to export and document this function; I'll leave that up to you.
 
-## NYTimes Books API
+## Secret management
 
-```{r include = FALSE}
-has_auth <- secret_has_key("HTTR2_KEY") && 
-  # Don't run in R CMD check on GHA because it hits NYTimes rate limits
-  !identical(Sys.getenv("GITHUB_WORKFLOW"), "R-CMD-check")
-knitr::opts_chunk$set(eval = has_auth)
-```
+We need to take a quick break from APIs to talk about secrets.
+Secrets are important, because every API (except for very simnple APIs like faker) is going to require that you identify yourself in some way typically with a username and password or an API key.
```

Comment:
```suggestion
Secrets are important, because every API (except for very simnple APIs like faker) is going to require that you identify yourself in some way, typically with an API key or a token.
```